contracts/Pool.sol

```diff
@@ -125,7 +126,7 @@ contract Pool is OwnableWithSuperAdmin, IPool {
         _stakerSlots[staker].add(_slotCurrent);
 
         _slotPendingBalance = _slotPendingBalance.add(value);
-        if (_slotPendingBalance == BEACON_AMOUNT) {
+        if (_slotPendingBalance >= BEACON_AMOUNT) {
```

Comment:
Add comment below that `>=` is just to be safe and the difference can be ignored